src/helpers/client/http-client.ts

```diff
@@ -48,7 +48,9 @@ class JSONRPCHttpClient {
       body: JSON.stringify(request),
     };
 
-    return fetch(asperaDesktop.globals.desktopUrl, options).then(response => {
+    const rpcServerURL = `${asperaDesktop.globals.desktopUrl}:${asperaDesktop.globals.rpcPort}`;
```

Comment:
This is fine. Could also create a method on globals that returns the url formatted.

Reply:
Done!